app/src/main/res/values/strings.xml

```diff
@@ -1,8 +1,22 @@
 <resources>
-    <string name="app_name">GeoQuiz</string>
-    <string name="true_button">True</string>
-    <string name="false_button">False</string>
-    <string name="question_australia">Canberra is the capital of Australia.</string>
-    <string name="correct_toast">Correct!</string>
-    <string name="incorrect_toast">Incorrect</string>
+        <string name="app_name">Test</string>
+        <string name="true_button">True</string>
+        <string name="false_button">False</string>
+        <string name="question_australia">Canberra is the capital of Australia.</string>
+        <string name="correct">Correct!</string>
+        <string name="incorrect">Incorrect!</string>
+        <string name="next_button">Next</string>
+        <string name="previous_button">Previous</string>
```

Comment:
remove the _button suffix and you will see that you have this string duplicated. Use just one in both places.
Remove the _button suffix in all cases